### ISSUE TITLE
Final RFC before 0.2: minor API tweaks

### ISSUE THREAD

**Issue by JDWarner** [scikit-fuzzy/scikit-fuzzy#98]
Up to the present, `scikit-fuzzy` has taken a NumPy-like approach: generally a functional interface, with essentially all functions brought into the base namespace. 

Benefits: allows a simple structure where people just `import skfuzzy as fuzz` and everything is available. Drawbacks: becomes cluttered quickly, reducing accessibility.

We're at the point where clutter becomes a concern. Specifically for the new control system interface, I think this may not be the right model. It may be best to keep it in `skfuzzy.control`. This allows all of the new functions to be grouped and more clearly linked. The downside is that users may not expect this, given the prior design, and have difficulty finding it. 

I'm leaning toward keeping the new API objects in `skfuzzy.control`, out of the base namespace, but would appreciate other thoughts. Also, would people prefer

``` python
import skfuzzy as fuzz
import skfuzzy.control as control  # Only available with this import

control.Antecedent
```

vs

``` python
import skfuzzy as fuzz

fuzz.control.Antecedent  # available but not in base `fuzz` namespace
```

**Comment by SophiaMitchell** [comment]
I think it makes sense to keep the new API in skfuzzy.control and out of the main namespace. As long as it's clear in the documentation where users can find it I don't think that will be too much of an issue, but it will keep things organized. As far as importing goes, I think the second way is most intuitive.

**Comment by JDWarner** [comment]
Thanks for your input, @SophiaMitchell

**Comment by JDWarner** [comment]
I agree, this is the ideal way forward.

**Comment by JDWarner** [comment]
Closed by f96fac4 in #105
